src/hotspot/share/runtime/stubRoutines.hpp

```diff
@@ -304,6 +304,9 @@ class StubRoutines: AllStatic {
     return dest_uninitialized ? _arrayof_oop_disjoint_arraycopy_uninit : _arrayof_oop_disjoint_arraycopy;
   }
 
```

Comment:
Could we have a comment here to note that 1) this method is implemented in architecture-specific code 2) any table that is returned must be allocated once-only in foreign memory rather generated in the code cache.

Reply:
Done!